octopus/src/test/scala/octopus/async/AsyncValidationSpec.scala

```diff
@@ -1,29 +1,26 @@
-package octopus
+package octopus.async
 
+import octopus.async.ToFuture.syntax._
 import octopus.dsl._
 import octopus.example.domain._
 import octopus.syntax._
-import octopus.{AsyncValidatorM => _}
+import octopus.{AppError, Fixtures, ValidationError, FieldPath, FieldLabel}
 import org.scalatest.{AsyncWordSpec, MustMatchers}
 
-import scala.concurrent.Future
+import scala.language.higherKinds
 
 abstract class AsyncValidationSpec[M[_]: ToFuture] extends AsyncWordSpec with Fixtures with MustMatchers {
 
-  implicit class FromMonadF[F[_]: ToFuture, A](value: F[A]){
-    def toFuture: Future[A] = implicitly[ToFuture[F]].toFuture(value)
-  }
-
-  implicit def app: AppError[M]
+  implicit def appError: AppError[M]
```

Comment:
Maybe move to constructor?

Reply:
It was there, but it was failing scala.js tests for `Future` due to different execution context than provided by `AsyncWordSpec`.